KATA_03/kata_03.js

```diff
@@ -1,2 +1,27 @@
 console.log("Kata 03");
 
+
+const students = [
+    { name: 'kalpana', age: 13, haveBeer: 'no' },
+    { name: 'aron', age: 20, haveBeer: 'no' },
+    { name: 'alex', age: 22, haveBeer: 'no' },
+    { name: 'eran', age: 23, haveBeer: 'no' },
+    { name: 'malika', age: 13, haveBeer: 'no' },
+    { name: 'anupama', age: 15, haveBeer: 'no' },
+    { name: 'dhanushka', age: 11, haveBeer: 'no' },
+    { name: 'madhushanka', age: 16, haveBeer: 'no' },
+    { name: 'thushan', age: 18, haveBeer: 'no' } 
+];
+
+function givePartyAccess(students) {
+    return students.map(student => {
```

Comment:
first do the filteration part. and secondly do map part. 

no need if statement inside the map()